spec/models/mapping_spec.rb

```diff
@@ -23,4 +23,5 @@
     mapping = Mapping.new(arena_record: record)
     expect(mapping.arena_record).to be_a(ArenaAddress)
   end
-end
\ No newline at end of file
+end
+end
```

Comment:
This appears to be an extra end.

Also, generally everything is nested under a single describe Maping block